src/Domain/DomainCreators/Shell.hpp

```diff
@@ -72,9 +73,16 @@ class Shell : public DomainCreator<3, TargetFrame> {
     static constexpr type default_value() noexcept { return false; }
   };
 
-  using options =
-      tmpl::list<InnerRadius, OuterRadius, InitialRefinement, InitialGridPoints,
-                 UseEquiangularMap, AspectRatio, UseLogarithmicMap>;
+  struct WhichPieces {
```

Comment:
Can this be `WhichWedges`? You describe it as choosing wedges, so using wedges here too would be more consistent. Same comment everywhere you use the word pieces.